content/_data/upgrades/2-361-1.adoc

```diff
@@ -0,0 +1,139 @@
+==== Jenkins requires Java 11 or newer
+
+Beginning with Jenkins 2.357, which was released on June 28, 2022, Jenkins requires Java 11 or newer on both the controller JVM and agent JVMs.
```

Comment:
Optional feedback for your consideration: I think we can avoid the mention of the weekly version number (i.e., 2.357) in this version of the content because this upgrade guide is exclusively written for users of the LTS line (i.e., 2.361.1). Since the intended audience is users of the LTS line, it might be more confusing than helpful for them to have to also read information about the weekly line. The information isn't wrong, just unnecessary, so I don't feel strongly either way about including or not including it.

Reply:
Thanks for confirming that! I was debating whether or not it was necessary, and agree that keeping it focused on the LTS release would be best. I'll remove this and update things accordingly.